common/src/main/java/com/wynntils/utils/Utils.java

```diff
@@ -4,16 +4,73 @@
  */
 package com.wynntils.utils;
 
+import java.awt.Toolkit;
+import java.awt.datatransfer.DataFlavor;
+import java.awt.datatransfer.StringSelection;
+import java.awt.datatransfer.Transferable;
+import java.awt.datatransfer.UnsupportedFlavorException;
 import java.util.Locale;
+import java.util.Random;
+import net.minecraft.Util;
 
 /**
  * This is a "high-quality misc" class. Helper methods that are commonly used throughout the project
  * without an aspect on minecraft can be put here. Keep the names short, but distinct.
  */
 public final class Utils {
     private static final Locale gameLocale = Locale.ROOT;
+    private static final Random random = new Random();
 
     public static Locale getGameLocale() {
         return gameLocale;
     }
+
+    public static Random getRandom() {
+        return random;
+    }
+
+    /**
+     * Open the specified URL in the user's browser if possible, otherwise copy it to the clipboard
+     * and send it to chat.
+     * @param url The url to open
+     */
+    public static void openUrl(String url) {
+        Util.getPlatform().openUri(url);
+    }
+
+    public static void copyToClipboard(String s) {
```

Comment:
The clipboard functions are not used anymore now with the new openUrl, but I assume you think they could be good to have for future uses..?

Reply:
I will just remove this for now. No dead code :\)